# HG changeset patch
# User Rik <rik@octave.org>
# Date 1424027376 28800
#      Sun Feb 15 11:09:36 2015 -0800
# Node ID bf6a909d3d11a621f5d99eb5a8db16528277304b
# Parent  1f66ff2a60a198d5ce093044ea84eefe69f70a68
doc: Use 'nonexistent' rather than 'non-existent' in documentation.

* external.txi, error_ids.m: Use 'nonexistent' rather than 'non-existent' in
documentation.

diff --git a/doc/interpreter/external.txi b/doc/interpreter/external.txi
--- a/doc/interpreter/external.txi
+++ b/doc/interpreter/external.txi
@@ -1088,17 +1088,17 @@ scope and will properly free memory with
 @subsection Input Parameter Checking in Oct-Files
 
 As oct-files are compiled functions they open up the possibility of
 crashing Octave through careless function calls or memory faults.
 It is quite important that each and every function have a sufficient level
 of parameter checking to ensure that Octave behaves well.
 
 The minimum requirement, as previously discussed, is to check the number
-of input arguments before using them to avoid referencing a non-existent
+of input arguments before using them to avoid referencing a nonexistent
 argument.  However, in some cases this might not be sufficient as the
 underlying code imposes further constraints.  For example, an external
 function call might be undefined if the input arguments are not
 integers, or if one of the arguments is zero, or if the input is complex
 and a real value was expected.  Therefore, oct-files often need additional
 input parameter checking.
 
 There are several functions within Octave that can be useful for the
diff --git a/scripts/miscellaneous/error_ids.m b/scripts/miscellaneous/error_ids.m
--- a/scripts/miscellaneous/error_ids.m
+++ b/scripts/miscellaneous/error_ids.m
@@ -29,17 +29,17 @@
 ## Indicates that a function was called with invalid input arguments.
 ##
 ## @item Octave:invalid-fun-call
 ## Indicates that a function was called in an incorrect way, e.g., wrong number
 ## of input arguments.
 ##
 ## @item Octave:invalid-indexing
 ## Indicates that a data-type was indexed incorrectly, e.g., real-value index
-## for arrays, non-existent field of a structure.
+## for arrays, nonexistent field of a structure.
 ##
 ## @item Octave:bad-alloc
 ## Indicates that memory couldn't be allocated.
 ##
 ## @item Octave:undefined-function
 ## Indicates a call to a function that is not defined.  The function may
 ## exist but Octave is unable to find it in the search path.
 ##
